### ISSUE TITLE
other dataset

### ISSUE THREAD

**Issue by xbysq** [ggsDing/Bi-SRNet#20]
Hello, I tried to test your proposed model on the Landsat-SCD dataset, but it didn't work well, I tried making the following changes:
num_classes = 5
ST_COLORMAP = [[255,255,255], [128,128,128], [130,87,87], [255,0,0], [0,0,255]]
ST_CLASSES = ['unchanged', 'farmland', 'desert', 'building', 'water']
The above modifications occur in : ./ datasets/RS_ST.py.
I don't know if you have done experiments on this dataset, can you give me a little comment?

**Comment by ggsDing** [comment]
Yes, I have run experiments on this dataset. Please refer to this [repository](https://github.com/ggsDing/SCanNet)

**Comment by xbysq** [comment]
Hello, I followed your part of the work, but the results of my training with Bi-SRNet are not very good, mIoU is only 81, did you make other parameter adjustments?

**Comment by ggsDing** [comment]
In the published paper the highest mIoU is 73.41. Did you do experiments with your own settings?

**Comment by xbysq** [comment]
>

**Comment by xbysq** [comment]
> In the published paper the highest mIoU is 73.41. Did you do experiments with your own settings?

Maybe I didn't make it clear what I meant, but I was talking about experiments on the Landsat dataset.

**Comment by xbysq** [comment]
> > In the published paper the highest mIoU is 73.41. Did you do experiments with your own settings?
> 
> Maybe I didn't make it clear what I meant, but I was talking about experiments on the Landsat dataset.

In the dataset mentioned in another work you gave, I found that the results of your training are very good, and I would like to ask if you have made some parameter adjustments.

**Comment by ggsDing** [comment]
> > > In the published paper the highest mIoU is 73.41. Did you do experiments with your own settings?
> > 
> > 
> > Maybe I didn't make it clear what I meant, but I was talking about experiments on the Landsat dataset.
> 
> In the dataset mentioned in another work you gave, I found that the results of your training are very good, and I would like to ask if you have made some parameter adjustments.

Please discuss under the [SCanNet](https://github.com/ggsDing/SCanNet) repository. The code will be made available soon.
